chromium_src/chrome/browser/ui/views/tabs/tab_style_views.cc

```diff
@@ -5,6 +5,8 @@
 
 #include "chrome/browser/ui/views/tabs/tab_style_views.h"
 #include "brave/browser/ui/color/brave_color_id.h"
+#include "brave/browser/ui/tabs/brave_tab_layout_constants.h"
+#include "brave/browser/ui/tabs/features.h"
 #include "brave/browser/ui/views/tabs/brave_tab_group_header.h"
 #include "brave/browser/ui/views/tabs/vertical_tab_utils.h"
 #include "chrome/browser/ui/views/tabs/tab_slot_controller.h"
```

Comment:
Are all these needed here?

Reply:
They are needed by `brave_tab_style_views.inc.cc`, which does not include its own dependencies (it relies on the includes defined in `tab_style_views.cc`).

Reply:
Yikes 😢